jupyterbook/content/code_gallery/data_access_notebooks/2017-03-21-ERDDAP_IOOS_Sensor_Map.ipynb

```diff
@@ -8,34 +8,20 @@
     "\n",
```

Comment:
<p>Consider rephrasing. Something like: ... Let's investigate the datasets we found and try to figure out why there is a discrepancy. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ioos/ioos_code_lab/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done!</p>